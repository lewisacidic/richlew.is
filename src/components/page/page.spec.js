import React from 'react'
import { render } from 'react-testing-library'

import Page from './page'

describe('Background', () => {
  it('works', () => {
    const { container } = render(<Page />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
