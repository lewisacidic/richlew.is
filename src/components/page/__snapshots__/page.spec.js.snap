// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Background works 1`] = `
.c0 {
  background: white;
  max-width: 960px;
  margin: auto;
  padding: 3rem;
  border-radius: 3rem;
}

.c0 > :last-child {
  margin-bottom: 0;
}

<div
  class="c0"
/>
`;
